pkg/cmd/util/clientconfig.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd"
+)
+
+func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
```

Comment:
add back the `// Copy of kubectl/cmd/DefaultClientConfig, using NewNonInteractiveDeferredLoadingClientConfig` comment so we have a breadcrumb to pick up upstream tweaks to the loading rules, etc